src/routes/quiz/biography/+page.svelte

```diff
@@ -97,13 +120,23 @@
 						>
 					{/if}
 				{:else}
-					<button
-						class="btn variant-filled-success w-fit"
-						type="button"
-						on:click={() => {
-							window.location.reload();
-						}}>Try Again</button
-					>
+				<div class="flex justify-between items-center w-full p-6">
+					{#if data.artist?.summary.split('<input').length === 1}
+						<input bind:value={guess} class="input px-2 w-48 m-2" name="answer" disabled={!!form} />
+					{/if}
+						{#if form?.correct}
+							<span class="w-3/4 font-bold text-success-500">{feedback}</span>
+						{:else}
+							<span class="w-3/4 font-bold text-error-500">{feedback} {form?.artist}</span>
+						{/if}
+						<button
```

Comment:
when using template strings use `` and ${}, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals?retiredLocale=de
```suggestion
							class=`btn w-fit ${form?.correct ? 'variant-filled-success' : 'variant-filled-error'}`
```